client/src/app/shared/components/media-upload-content/media-upload-content.component.ts

```diff
@@ -140,17 +140,7 @@ export class MediaUploadContentComponent extends BaseModelContextComponent imple
 
         // detect changes in the form
         this.directorySelectionForm.valueChanges.subscribe(formResult => {
-            // undefined == none selection.
-            if (formResult.directoryId || formResult.directoryId === undefined) {
-                this.directoryId = formResult.directoryId || null;
-                let url = `/${this.activeMeetingId}/mediafiles/upload`;
-                if (this.directoryId) {
-                    url += `/${this.directoryId}`;
-                }
-                this.router.navigate([url], {
-                    replaceUrl: true
-                });
-            }
+            this.directoryId = formResult.directoryId;
```

Comment:
Quite an old artifact. While we are at it, please push the `Subscription` into the subscription array so it can be freed. Can be merged after that